snow/consensus/snowman/poll/early_term_traversal.go

```diff
@@ -11,38 +11,37 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 
 	"github.com/ava-labs/avalanchego/ids"
-	"github.com/ava-labs/avalanchego/snow"
 	"github.com/ava-labs/avalanchego/utils/bag"
 )
 
 var (
 	errPollDurationVectorMetrics = errors.New("failed to register poll_duration vector metrics")
 	errPollCountVectorMetrics    = errors.New("failed to register poll_count vector metrics")
 
-	terminationReason    = "reason"
-	exhaustedReason      = "exhausted"
-	earlyFailReason      = "early_fail"
-	earlyAlphaPrefReason = "early_alpha_pref"
+	terminationReason = "reason"
+	exhaustedReason   = "exhausted"
+	earlyFailReason   = "early_fail"
+	earlyAlphaReason  = "early_alpha"
```

Comment:
We don't differentiate between `pref` and `conf` anymore, so seems like we should just unify the name.